.vscode/settings.json

```diff
@@ -6,5 +6,7 @@
   "[typescript]": {
     "editor.formatOnSave": false
   },
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "prettier.printWidth": 180,
```

Comment:
I don't think you should use prettier, and you shouldn't set a rule to limit line lengths.  Much better to just rely on vscode and (es/te)lint.

It is your choice...but I really don't want to deal in PRs with prettier reformating...